stable/artifactory/templates/artifactory-statefulset.yaml

```diff
@@ -382,6 +387,11 @@ spec:
         - name: installer-info
           mountPath: "/artifactory_bootstrap/info/installer-info.json"
           subPath: installer-info.json
+      {{- if .Values.artifactory.persistence.googleStorage.gcpServiceAccount.enabled }}
+        - name: gcpcreds-json
+          mountPath: "/artifactory_bootstrap/gcp.credentials.json"
```

Comment:
If I’m not mistaken, this directory is only used on the initial startup. This is not a good place for this file because users might be co fused when the file is not used. If you want to keep it here, I suggest you add the copyOnEveryStartup to this example

Reply:
I just copied the code exactly from the artifactory-ha chart. If its to be changed, it should be kept consistent across both

Reply:
👍